ibis/backends/snowflake/registry.py

```diff
@@ -187,13 +187,6 @@ def _extract_url_query(t, op):
     ops.NotExistsSubquery,
     # ibis.expr.operations.maps
     ops.MapKeys,
-    # ibis.expr.operations.numeric
```

Comment:
The tests for numeric operations were expected to be in the `test_numeric.py` file, so that's where the problem came from. I'm moving it now.

Reply:
Can you elaborate a bit on what you mean? This list of ops shouldn't be related to where we happen to be testing them.

Reply:
I think I understand. Were you only running a subset of the tests?

Reply:
I run tests multiple times. Each time I run it, I remove a different operation, then run a subset of tests depending on the name of the module in which the operation is.

My heuristic is similar to the code below.
```python
ops = [
    ...
    'ibis.expr.operations.analytic.CumeDist',
    ...
]
....
for op in ops
  if '.numeric.' in op:
      test_files = list(map(str, Path("ibis/backends/tests/").glob("test_numeric.py")))
  elif '.strings.' in op:
      test_files = list(map(str, Path("ibis/backends/tests/").glob("test_string.py")))
  elif '.arrays.' in op:
      test_files = list(map(str, Path("ibis/backends/tests/").glob("test_array.py")))
  elif '.reductions.' in op:
      test_files = list(map(str, Path("ibis/backends/tests/").glob("test_agg*.py")))
  else:
      test_files = list(map(str, Path("ibis/backends/tests/").glob("test_*.py")))
```
We don't have any problem in the Ibis core, but it was a mistake on my side.